.eslintrc.js

```diff
@@ -19,8 +19,10 @@ module.exports = {
     "import"
   ],
   "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
I generally like having this as the exception but I see that would add a ton of disable line rules. Wish there was some way to say "in these modules".

Reply:
I forgot about the top of the file one, maybe that would work better if we want to keep things generally strict.

Reply:
the no-implicit-any rule makes sense to me, but explicitly specifying `any` is sometimes a necessary escape hatch, and seeing the explicit `any` type is already like a giant neon sign that something weird is going on in the code. i always stop and think twice whenever i see `any` on a PR changes, i just don't think the linter rule is providing any value there